docs/nydus-fscache.md

```diff
@@ -89,13 +89,10 @@ make
 mkdir -p /var/lib/containerd-nydus
 
 ./bin/containerd-nydus-grpc \
- --config-path /etc/nydus/nydusd-config.fscache.json \
+ --nydusd-config /etc/nydus/nydusd-config.fscache.json \
  --daemon-mode shared \
```

Comment:
Is it necessary?

Reply:
You are right, we can also remove the flag as `--fs-driver fscache` enforces it.

Reply:
fixed